olivertwist/main.py

```diff
@@ -33,8 +34,9 @@
     help="Do/Don't open HTML report in browser. Implies --html",
 )
 def main(input, html=True, browser=False):
+    optional_config = ConfigParser.from_default_path()
     manifest = Manifest(json.load(input))
-    rule_engine = RuleEngine.with_default_rules()
+    rule_engine = RuleEngine.with_default_rules(config = optional_config)
```

Comment:
PEP8 Style - remove spaces around the equals symbol for keyword args:

`rule_engine = RuleEngine.with_default_rules(config=optional_config)`